src/travertino/node.py

```diff
@@ -14,6 +18,71 @@ def __init__(self, style, applicator=None, children=None):
             for child in children:
                 self.add(child)
 
+    @property
+    def style(self):
+        """The node's style.
+
+        Assigning a style triggers an application of that style if an applicator has
+        already been assigned.
+        """
+
+        return getattr(self, "_style", None)
```

Comment:
Pretty sure we can just use `self._style` here; there's no path where it can be accessed by a user before it is set.